src/locales/ko/mystery-encounters/the-pokemon-salesman-dialogue.json

```
@@ -1,23 +1,23 @@
 {
-  "intro": "A chipper elderly man approaches you.",
-  "speaker": "Gentleman",
-  "intro_dialogue": "Hello there! Have I got a deal just for YOU!",
-  "title": "The Pokémon Salesman",
-  "description": "\"This {{purchasePokemon}} is extremely unique and carries an ability not normally found in its species! I'll let you have this swell {{purchasePokemon}} for just {{price, money}}!\"\n\n\"What do you say?\"",
-  "description_shiny": "\"This {{purchasePokemon}} is extremely unique and has a pigment not normally found in its species! I'll let you have this swell {{purchasePokemon}} for just {{price, money}}!\"\n\n\"What do you say?\"",
-  "query": "What will you do?",
+  "intro": "건장한 노신사가 다가왔다.",
```

```suggestion
  "intro": "쾌활한 노신사가 다가왔다.",
```
chipper의 의미가 heerful and lively라고 되어 있어서 쾌활 정도가 어떨까 합니다..